Updates to the 'Importing modules into Pantheon' module (#450)
redhataccess/pantheon: pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc_importing-content.adoc

[id="importing-(ContentTerm}_{context}"]

= Importing {ContentTerm}

[role="_abstract"]
You can import modules and assemblies into {ProductShortName} from either GitHub or GitLab.

.Prerequisites

//* You are assigned a publisher or author role in {ProductShortName}.
* You are logged in.
* You obtained the Git repository URL for the {ContentTerm} you want to import (For example, https://github.com/openshift/openshift-docs).

.Procedure

. Navigate to the link:{LinkToGitImportPage}[Git Import page].
. Enter the Git repository URL for the {ContentTerm} you want to import.

> . By default, this procedure imports the "master" branch of the {ContentTerm} repository, but you can enter a specific branch to import instead if you prefer.

Reviewer: I don't understand this line. How do you import master? How do you import a branch?

. Click *Submit*.
. Click *Yes* to confirm and return to the link:{LinkToSearchPage}[Search page], or *No* to remain on the Git Import page.
+

> Note that it takes {ProductShortName} some time to synchronize its database and the UI following the import. Therefore your files may not be immediately available when you are redirected in the UI,

Reviewer: ```suggestion
Note that {ProductShortName} takes time to synchronize its database and the GUI following the import. Therefore your files might not be immediately available when you are redirected in the GUI,
```
Looking for more brevity. Also, don't use "may", it's ambiguous.
